PolyEngine/Core/Src/EnumUtils.hpp

```diff
@@ -64,7 +65,8 @@ namespace Poly {
 			friend class EnumArray<T, E, SIZE>;
 		};
 
-		EnumArray() = default;
+		//EnumArray() = default;
+		EnumArray() : Data{} {}
```

Comment:
Yeah, it will default-initialize them, which (for pointers) is doing nothing (as per http://en.cppreference.com/w/cpp/language/default_initialization).